pkg/cmd/target/target.go

```diff
@@ -240,7 +251,19 @@ func (o *TargetOptions) Complete(f util.Factory, cmd *cobra.Command, args []stri
 }
 
 // Validate validates the provided options
-func (o *TargetOptions) Validate() error {
+func (o *TargetOptions) Validate(f util.Factory) error {
```

Comment:
You broke the interface contract with this change. Validate does not have args!!! Everthing you need for validation needs to be added to the options in complete. That's the idea.